packages/sakuli-core/package.json

```diff
@@ -37,6 +43,7 @@
     "class-validator": "^0.9.1",
     "common-tags": "^1.8.0",
     "date-fns": "^1.29.0",
+    "fs-extra": "^7.0.1",
```

Comment:
Is it intended that `fs-extra` is a dev dependency? I thought the initial problem was, that it is not bundled in the prod package? :thinking: 

BTW: How did you know, that it was missing in the core? I searched for fs-extra in the whole project but was unable to find it. Therefore I had no clue where it should be added.

Reply:
`fs-extra` is used in https://github.com/sakuli/sakuli/blob/65efa9b65312f782d2fb8c0f82b2b656ee68ca99/packages/sakuli-core/src/bootstrap/load-bootstrap-options.function.ts#L3